drivers/dide/inst/templates/provision.sh

```diff
@@ -5,10 +5,9 @@ echo generated on date: {{date}}
 echo hipercow version: {{hipercow_version}}
 echo running on: $(hostname -f)
 
-export PATH=/opt/apps/lmod/lmod/libexec:$PATH
-source /opt/apps/lmod/lmod/init/bash
-export LMOD_CMD=/opt/apps/lmod/lmod/libexec/lmod
-module use /modules-share/modules/all
+source /etc/profile
+
+module use /modules/modules/all
```

Comment:
This change is because it turns out lmod installation is much simpler than I thought. infra-scripts does it for us elsewhere, and sourcing /etc/profile will set the environment varialbes we need.

The other change is a bit random - I reinstalled all the modules yet again to try and get them working better, and seemed to call it modules instead of modules-share.